sentry_sdk/integrations/trytond.py

```diff
@@ -0,0 +1,35 @@
+import sentry_sdk
+import sentry_sdk.integrations
+
+from trytond.exceptions import TrytonException  # type: ignore
+from trytond.wsgi import app  # type: ignore
+
+
+# TODO: trytond-worker, trytond-cron and trytond-admin intergations
+
+
+class TrytondWSGIIntegration(sentry_sdk.integrations.Integration):
```

Comment:
If this is a WSGI app then it might be useful to wrap it in the WSGI middleware: https://docs.sentry.io/platforms/python/wsgi/

This will add request data to each crash

Reply:
In such case, I think it will be good to prevent to store the password from the request date when the login endpoint if called.

Reply:
Sentry removes keys called password by default, does that work for you?

Reply:
Good, it is the name used by Tryton.

Reply:
Yeah that should work. Our tech for data protection tech is something that is beyond integrations anyway (using `before_send` and serverside datascrubbing): https://docs.sentry.io/error-reporting/configuration/filtering/

Reply:
I'll check it out and I'll also assert that passwords do not get leaked.

But this also raises a concern about leaking bearer authorization headers, the default auth mechanism for Tryton APIs. Can we set the integration to skip those by default?

Also, Tryton has a pluggable auth system so you can easily tweak the login. It can be used for 2FA with temporary secrets, but the default password login can be completely overridden by other systems that could request secrets in arbitrary keys (am I wrong, @cedk ?). I'd vote for completely masking request/response bodies for the Tryton login endpoint. Is that feasible?

Reply:
Yes the authentication mechanism can be completely customized. But I do not see why someone would use another key than 'password' to receive a secret password.
For temporary token like TOTP, I do not see big issue to get them logged as they are temporary.
But of course, if it is even better to not log any data from the login endpoint if it is doable.

Reply:
Done. Please give it a look.

It does filter out the password field and as @cedk said, one time secrets shouldn't be a concern and other login systems shouldn't be using a field different than password

Reply:
lgtm